kuma/landing/templates/landing/promote_buttons.html

```diff
@@ -284,7 +284,7 @@
         // Update the button code.
         $('#button-code').val([
             '<a href="', sel.tracked_url, '" title="', sel.alt_text, '">',
-            '<img src="{{ request.build_absolute_uri(MEDIA_URL) }}img/promote/promobutton_', sel.id, '.png" ', 
+            '<img src="{{ request.build_absolute_uri(STATIC_URL) }}img/promote/promobutton_', sel.id, '.png" ',
```

Comment:
The use of `request.build_absolute_uri` isn't needed here since the staticfiles storage should handle such decisions. Also same case of using the `static` tag here is needed